channels/tsmf/client/tsmf_media.c

```diff
@@ -246,22 +284,38 @@ static BOOL tsmf_stream_process_ack(void* arg, BOOL force)
 	BOOL rc = FALSE;
 
 	if (!stream)
-		return FALSE;
+		return TRUE;
 
```

Comment:
I think this is correct, it matches my description for the return code. I use TRUE to indicate that no samples are currently available, which would be the case if there was no stream. Make sense at all?